Example/Tests/Tests.m

```diff
@@ -169,7 +169,7 @@
 
             SEGScreenPayload *payload = [[SEGScreenPayload alloc] initWithName:@"Shirts" properties:@{} context:@{} integrations:@{}];
             [integration screen:payload];
-            [verify(amplitude) logEvent:@"Loaded a Screen" withEventProperties:@{}];
+            [verify(amplitude) logEvent:@"Loaded a Screen" withEventProperties:@{ @"name" : @"Shirts" }];
```

Comment:
@ladanazita I noticed this in the Android PR too, but we should also add a test case to make sure any custom properties are sent through + `name`.